src/Game/UI/Gumps/ContainerGump.cs

```diff
@@ -89,6 +91,8 @@ public ContainerGump(uint serial, ushort gumpid, bool playsound) : base(serial,
 
         public ushort Graphic { get; }
 
+        public uint ParentSerial { get; set; }
```

Comment:
probably it's better to set the property as readonly. Removing the `set` is enough

Reply:
Removed.